agent/consul/authmethod/kubeauth/k8s.go

```diff
@@ -20,7 +21,7 @@ import (
 
 func init() {
 	// register this as an available auth method type
-	authmethod.Register("kubernetes", func(method *structs.ACLAuthMethod) (authmethod.Validator, error) {
+	authmethod.Register("kubernetes", func(_ hclog.Logger, method *structs.ACLAuthMethod) (authmethod.Validator, error) {
```

Comment:
can you eli5 what is happening here? i am confused

Reply:
nevermind i looked down disregard

Reply:
`_` is the blank identifier. Just like `_, err := blah()`. The body of the function doesn't use the argument.